driver-core/src/main/com/mongodb/event/ConnectionPoolListener.java

```diff
@@ -76,15 +114,46 @@ default void waitQueueExited(ConnectionPoolWaitQueueExitedEvent event) {
      * Invoked when a connection is added to a pool.
      *
      * @param event the event
+     * @deprecated Prefer {@link #connectionCreated}
      */
+    @Deprecated
```

Comment:
I can't tell but it looks like this event is no longer called at all. So we should either deprecate and continue support or remove it and add a note to the upgrade guide that its been renamed.

Reply:
@jyemin I see your point. Then we keep the deprecated methods until version 5.

Reply:
I haven't been able to think of anything more elegant.